<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_complain">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пожаловаться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reputation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Репутация всех пользователей</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_reputation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>201</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_allComplaintsForUser">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Посмотреть все жалобы на 
конкретного пользователя</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_allComplaintsForUser">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>201</y>
      <width>311</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_toMain">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>550</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>На главную</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
